packages-exp/auth-exp/src/model/public_types.ts

```diff
@@ -358,7 +358,7 @@ export interface Persistence {
  * Interface representing ID token result obtained from {@link User.getIdTokenResult}.
  *
  * @remarks
- * It contains the ID  token JWT string and other helper properties for getting different data
+ * It contains the ID token JWT string and other helper properties for getting different data
```

Comment:
Should this be "The result" or something more clear than "It?"

Reply:
It's specifically this interface so I just put the name, in backticks?